src/engine/ruleset/decoders/redis/redis.yml

```diff
@@ -0,0 +1,110 @@
+---
+name: decoder/redis/0
+
+metadata:
+  module: redis
+  title: Redis logs decoder
+  description: Decoder for Linux redis logs
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://docs.rs/redis-module/latest/redis_module/logging/index.html
+
+# Example redis logs:
+
+# Darwin
+# 4961:M 30 May 12:50:13.457 * Increased maximum number of open files to 10032 (it was originally set to 4864).
+#                 _._
+#            _.-``__ ''-._
+#       _.-``    `.  `_.  ''-._           Redis 3.0.2 (00000000/0) 64 bit
+#   .-`` .-```.  ```\/    _.,_ ''-._
+#  (    '      ,       .-`  | `,    )     Running in standalone mode
+#  |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
+#  |    `-._   `._    /     _.-'    |     PID: 4961
+#   `-._    `-._  `-./  _.-'    _.-'
+#  |`-._`-._    `-.__.-'    _.-'_.-'|
+#  |    `-._`-._        _.-'_.-'    |           http://redis.io
+#   `-._    `-._`-.__.-'_.-'    _.-'
+#  |`-._`-._    `-.__.-'    _.-'_.-'|
+#  |    `-._`-._        _.-'_.-'    |
+#   `-._    `-._`-.__.-'_.-'    _.-'
+#       `-._    `-.__.-'    _.-'
+#           `-._        _.-'
+#               `-.__.-'
+# 4961:M 30 May 12:50:13.463 # Server started, Redis version 3.0.2
+
+# Debian
+# 30 May 10:04:45 . 0 clients connected (0 slaves), 618932 bytes in use, 0 shared objects
+
+# Windows
+# [2932] 31 May 04:32:08 - 0 clients connected (0 slaves), 1179968 bytes in use
+
+parse:
+  logpar:
+    # 26571:M 27 Dec 2018 11:19:18.874 * Synchronization with replica 10.114.208.18:6023 succeeded
+    - event.original: <process.pid>:<~log.role> <@timestamp/%d %b %Y %T> <log.level> <message>
+
+    # Darwin
+    - event.original: <process.pid>:<~log.role> <@timestamp/%d %b %T> <log.level> <message>
+
+    # Debian
+    - event.original: <@timestamp/%d %b %T> <log.level> <message>
+
+    # Windows
+    - event.original: <~/ignore/[><~log.pid>] <@timestamp/%d %b %T> <log.level> <message>
```

Comment:
Could the example log be added above each logpar expression to make it easier to read?